docs/content/latest/manage/upgrade-deployment.md

```diff
@@ -22,11 +22,21 @@ If you plan to script this in a loop, then a pause of about 60 seconds is recomm
 Your data/log/conf directories are generally stored in a separate location which stays the same across the upgrade so that the cluster data, its configuration settings are retained across the upgrade.
 {{< /tip >}}
 
+## Install new version of YugabyteDB
+First we need to install the new version of YugabyteDB in a new location. 
+For Centos this would be something like:
+```
+1. wget https://downloads.yugabyte.com/yugabyte-$VER.tar.gz
+2. tar xf yugabyte-$VER.tar.gz -C /home/yugabyte/softwareyb-$VER/ 
+3. cd /home/yugabyte/softwareyb-$VER/ 
+4. ./bin/post_install.sh
+```
+
 ## Upgrade YB-Masters
 
 ```
 1. pkill yb-master
-2. switch binaries to new release
+2. make sure we're on the dir of the new version (cd /home/yugabyte/softwareyb-$VER/) 
 3. start the yb-master process
```

Comment:
start the yb-master process
-->
start (the newer version of) the yb-master process

similar change to to yb-tserver section